Header header

uint8 radarNum
uint8 numObjects
float64[] accel_x
float64[] vel_x
float64[] pos_x
float64[] pos_y
float64[] existProb
bool[] valid
